var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -89,6 +89,9 @@ class Vtk(CMakePackage):
     depends_on('mpi', when='+mpi')
 
     depends_on('expat')
+    # See <https://gitlab.kitware.com/vtk/vtk/-/issues/18033> for why vtk doesn't
+    # work yet with freetype 2.10.3 doesn't work yet (including possible patches)
```

Comment:
Suggest removing redundant verbiage, such as:

```suggestion
    # work yet with freetype 2.10.3 (including possible patches)
```